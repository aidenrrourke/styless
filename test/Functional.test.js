import React from 'react';
import renderer from 'react-test-renderer';
import styled from 'styled-components';
import 'jest-styled-components';

test( 'Darkens the primary color 20%', () => {
	const Div = styled.div`
		color: darken(@primary, 20%);
	`;
	const tree = renderer.create( <Div primary="red"/> ).toJSON()
	expect( tree ).toMatchSnapshot();
} );