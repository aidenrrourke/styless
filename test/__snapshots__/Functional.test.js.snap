// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Darkens the colors in linear-gradient 1`] = `
.c0 {
  background: linear-gradient(#f66 0%,#009 100%);
}

<div
  className="c0"
  end="blue"
  start="red"
/>
`;

exports[`Darkens the primary color 20% 1`] = `
.c0 {
  color: #900;
}

<div
  className="c0"
/>
`;
